addons/binding/org.openhab.binding.dscalarm/ESH-INF/thing/keypad.xml

```diff
@@ -51,6 +51,14 @@
 				<label>Keypad AC LED</label>
 				<description>Keypad AC LED (0=Off, 1=On, 2=Flashing)</description>
 			</channel>
+            <channel id="keypad_lcd_update" typeId="message">
```

Comment:
Hello @kaikreuzer,

Doing an auto format across the entire project seems to ignore the .xml files, so I had to do it in the XML editor.  Also, I noticed the formatter profile ESH, specifies spaces for indention as seen here:

![esh](https://cloud.githubusercontent.com/assets/7375448/20238398/9b3ec120-a8b8-11e6-9f97-23cacce6596e.JPG)

Evidently Eclipse treats .xml files differently when it comes to formatting, because it replaced the spaces with tabs in the XML editor.  All the java code has spaces for indention because of the formatter profile.  Is that the way it is supposed to be?